docs/compass/03-automatic-scenario-assignment.md

```diff
@@ -0,0 +1,213 @@
+# Automatic Scenario Assignment
+
+To connect and group your Applications and Runtimes, assign them to the same scenario.
+Automatic Scenario Assignment feature allows defining condition when a Scenario is automatically assigned to the Runtime.
+
+## API
+
+AutomaticScenarioAssignment is defined in the following way:
+```graphql
+type AutomaticScenarioAssignment {
+   scenarioName: String!
+   selector: Label!
+}
+
+type Label {
+   key: String!
+   value: Any!
+}
+
+```
+If the Runtime is labeled with a label that matches the label's selector, the Runtime is assigned to the given Scenario.
+
+
+Director API contains the following mutations for managing Automatic Scenario Assignments.
+```graphql
+   createAutomaticScenarioAssignment(in: AutomaticScenarioAssignmentSetInput!): AutomaticScenarioAssignment 
+   deleteAutomaticScenarioAssignmentForScenario(scenarioName: String!): AutomaticScenarioAssignment 
+   deleteAutomaticScenarioAssignmentsForSelector(selector: LabelSelectorInput!): [AutomaticScenarioAssignment!]! 
+}
+```
+When creating Assignment, the following conditions are checked:
+* for given scenario, at most one Assignment exists
+* given scenario has to exists
+* selector's value type is a string
+
+Director API contains queries for fetching all assignments, assignments for the given Scenario and the given label selector.
```

Comment:
I would recommend adding subheaders:
`### Queries`, `### Mutations` etc.

Reply:
I agree this might be helpful.